baybe/utils/plotting.py

```diff
@@ -0,0 +1,56 @@
+"""Plotting utilities."""
+
+import json
+import os
+import warnings
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+
+
+def create_plots(data: pd.DataFrame, path: Path, base_name: str, **kwargs) -> None:
+    """Create plots from a given data frame and save them as a svg file.
+
+    The plots will be saved in the location specified by ``path``.
+    The attribute ``base_name`` is used to define the name of the outputs.
+
+    If the ``SMOKE_TEST`` variable is set, no plots are being saved.
+
+    Using the ``BAYBE_MULTIVERSION_PLOTS`` environment variable, it is possible to
+    create plots for the light and dark version of the documentation. If this variable
+    is not set, a single file named ``{base_name}_check.svg`` is created.
+
+    Args:
+        data: The data frame containing the data to be plotted.
+        path: The path of the file that should be saved.
```

Comment:
The description does not match with what the variable really does. In your implementation, it refers do the directory, but you say here that it refers to the file.

Reply:
Adjusted.